simulators/extras/ArgumentsAndCredentialsHandler.py

```diff
@@ -176,10 +179,11 @@ def RemoveTrailingSlashFromBaseUrl(baseUrl):
     return baseUrl
 
 
-def CheckTenantConnection(baseUrl, C8Y_HEADERS):
+def CheckTenantConnection(baseUrl, C8Y_HEADERS, session):
```

Comment:
Why is this session added here? This function is also used by the export script. Please adapt

Reply:
Or set a default

Reply:
the session is also used to verify the connection; I added a session now also to the ExportProfileData.py 
- unfortunately I didnt find a way to use the session or the VERIFY_SSL_CERTIFICATE variable in the c8y_api to also allow exporting without verifying the SSL certificate